.gitignore

```diff
@@ -27,3 +27,6 @@ configs/config.yaml
 
 # outputs from build
 Containerfile
+
+# Ignore kafka data
+kafka/data
```

Comment:
when i make kafka-up i get an error that this directory doesn't exist, maybe the make file should create it?

Reply:
Thanks! I fixed this by checking and creating the directory if necessary.